examples/compose/vtcompose/vtcompose.go

```diff
@@ -311,7 +311,7 @@ func getTableName(sqlFile string) string {
 		log.Fatalf("reading sqlFile file %s: %s", sqlFile, err)
 	}
 
-	r, _ := regexp.Compile("CREATE TABLE ([a-z_-]*) \\(")
```

Comment:
Expression should not consider dash (-) to be a valid character in a table name.